nautobot_netbox_importer/management/commands/import_netbox_objectchange_json.py

```diff
@@ -0,0 +1,105 @@
+"""Definition of "manage.py import_netbox_objectchange_json" Django command for use with Nautobot."""
+import argparse
+import json
+
+from packaging import version
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.contenttypes.models import ContentType
+
+from nautobot.extras.models import ObjectChange
+
+from nautobot_netbox_importer.utils import ProgressBar
+from nautobot_netbox_importer.command_utils import validate_netbox_version, initialize_logger
+from nautobot_netbox_importer.diffsync.models.validation import netbox_pk_to_nautobot_pk
+
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """Implementation of import_netbox_objectchange_json command."""
+
+    help = "Import ObjectChange objects from NetBox YAML data dump into Nautobot's database"
+
+    def add_arguments(self, parser):
+        """Add parser arguments to the import_netbox_objectchange_json management command."""
+        parser.add_argument("json_file", type=argparse.FileType("r"))
+        parser.add_argument("objectchange_json_file", type=argparse.FileType("r"))
+        parser.add_argument("netbox_version", type=version.parse)
+        parser.add_argument("--dryrun", type=bool, default=False)
+
+    def handle(self, *args, **options):
+        """Handle execution of the import_netbox_json management command."""
+
+        def process_objectchange(entry):
```

Comment:
To facilitate the creation of unit test automation, it would probably be good for this function to exist on its own rather than only within the `handle()` method, so that it can be called independently.